### ISSUE TITLE
chore: Apply for netlify free

### ISSUE THREAD

**Issue by gaocegege** [tensorchord/envd-docs#209]
https://opensource-form.netlify.com/

**Comment by gaocegege** [comment]
![image](https://user-images.githubusercontent.com/5100735/207772618-8f02c45b-53eb-4b16-89a5-ed837e5ff5bb.png)


I do not know how to add a centered img in the vitepress footer, needs some help from frontend developers.

**Comment by gaocegege** [comment]
```
    footer: {
      message: 'Released under the Apache-2.0 License. Built with VitePress.',
      copyright: 'Copyright © 2022 TensorChord, Inc. <p><a href="https://www.netlify.com"> <img src="https://www.netlify.com/v3/img/components/netlify-color-accent.svg" alt="Deploys by Netlify" /> </a></p>'
    }
```
